### ISSUE TITLE
[New] Simplify legend in colored maps

### ISSUE THREAD

**Issue by Inglezos** [lisphilar/covid19-sir#747]
## Summary of this new feature
Use horizontal legend in colored maps.

## (Optional) Solution with the current version

```
cax = divider.append_axes("bottom", size="5%", pad=0.1)
plot_kwargs["legend_kwds"] = {'orientation': "horizontal"}
```

**Comment by lisphilar** [comment]
A the next step, could you add legend label to indecate colors and values are in log-scale?
https://gist.github.com/lisphilar/cc5599d974fe5b450d8d9bb519628009

**Comment by lisphilar** [comment]
With #746 and #772,
```Python
import covsirphy as cs
data_loader = cs.DataLoader()
pcr_data = data_loader.pcr()
country = "Italy"
pcr_data.map(country)
```
![pcr_japan](https://user-images.githubusercontent.com/7270139/118355543-7e6c6480-b5ab-11eb-88f8-c30220fc5e64.jpg)
